src/installer/bin.ts

```diff
@@ -13,6 +13,9 @@ debug(`INIT_CWD: ${process.env.INIT_CWD}`)
 // huskyDir is ONLY used in dev, don't use this arguments
 const [, , action, huskyDir = path.join(__dirname, '../..')] = process.argv
 
+// Determine whether environment is CI
+const { isCI } = ci.isCI
```

Comment:
Except if I'm missing something, shouldn't it be `const { isCI } = ci` based on https://github.com/watson/ci-info#usage?

Reply:
Based on the library's usage documentation, `ci` is the encapsulating namespace of the library, and `isCI` is the method that actually shows whether or not the call was invoked in a CI environment or not. Not sure if that answers your question, I'll keep an 👀out for further comments! Also, sorry for the late response!

Reply:
```suggestion
const { isCI } = ci
```

Looks correct to me based on the linked documentation.

Reply:
@typicode @fearphage Updated - thank you both for the replies! 😄